recipe/conda_build_config.yaml

```diff
@@ -240,7 +240,7 @@ pin_run_as_build:
   librdkafka:
     max_pin: x.x.x
   librsvg:
-    max_pin: x.x
+    max_pin: x
```

Comment:
I was going for a `x.x` b/c we don't really have the ABI info on version `2.43`. There is no reason to doubt that `x` is good enough but I prefer to play on the safe side though.

What do you think @isuruf?

Reply:
I looked at the symbols and 2.44 added a bunch of symbols compared to 2.40 (I think in part due to the switch to rust) and 4 symbols removed.

```
$diff -ru 2.40.txt 2.44.txt  | grep -E "^\-"
--- 2.40.txt    2019-07-22 09:38:07.325532888 -0500
-rsvg_cairo_to_pixbuf
-rsvg_css_parse_color
-rsvg_pixbuf_from_data_with_size_data
```

According to https://launchpad.net/debian/+source/librsvg/+changelog these should have been private and no package in debian links to them.

Reply:
So `x.x` then?

Reply:
I was making the case for `x` above. :smile:

Reply:
If you think it is stable enough then sure! Let's go for it!